flow/Stats.actor.cpp

```diff
@@ -45,21 +45,32 @@ double Counter::getRate() const {
 }
 
 double Counter::getRoughness() const {
-	double elapsed = now() - interval_start;
+	double elapsed = now() - roughness_interval_start;
 	if(elapsed == 0) {
 		return 0;
 	}
 
+	// If we have time samples t in T, and let:
+	// n = size(T) = interval_delta
+	// m = mean(T) = elapsed / interval_delta
+	// v = sum(t^2) for t in T = interval_sq_time
+	// 
+	// The formula below is: (v/(m*n)) / m - 1
+	// This is equivalent to (v/n - m^2) / m^2 = Variance(T)/m^2
```

Comment:
Shouldn't the variance of T calculated as this?

<img src="https://render.githubusercontent.com/render/math?math=\sum_{i=1}^n%20\frac{(t_i - m)^2}{n-1}">

Reply:
These are equivalent (see https://en.wikipedia.org/wiki/Variance#Sample_variance), except that yours is also correcting for bias by dividiving by `n-1`.